repos/system_upgrade/common/actors/checkdetecteddevicesanddrivers/libraries/checkdddd.py

```diff
@@ -35,6 +35,16 @@ def create_inhibitors(inhibiting_entries):
                     source=get_source_major_version(),
                 )
             ),
+            reporting.ExternalLink(
```

Comment:
I would keep just one link in the tooling and possibly merge content of both solution.